CHANGELOG.md

```diff
@@ -1,20 +1,22 @@
 # Lotus changelog
 
-# v1.28.0 / 2024-07-23
-This is the MANDATORY Lotus v1.28.0 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇.
+# v1.28.1 / 2024-07-24
 
-**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.** 
+This is the MANDATORY Lotus v1.28.1 release, which will deliver the Filecoin network version 23, codenamed Waffle 🧇. v1.28.1 is also the minimal version that supports nv23.
+**This release sets the Mainnet to upgrade at epoch 4154640, corresponding to 2024-08-06T12:00:00Z.**
 
 ## ☢️ Upgrade Warnings ☢️
 - If you are running the `v1.26.x` version of Lotus, please go through the `Upgrade Warnings` section for the `v1.27.*` releases, before upgrading to this RC.
-
+- Note that v1.28.0 misses a critical feature removal and a bug fix and was retracted. Please update your node to v1.28.1 or above before the nv23 upgrade!
 - This upgrade includes an additional migration to the events database. Node operators running Lotus with events turned on (off by default) may experience some delay in initial start-up of Lotus as a minor database migration takes place. See [filecoin-project/lotus#12080](https://github.com/filecoin-project/lotus/pull/12080) for full details.
+- ALL LEGACY PRECOMMITS MUST BE PROVECOMMITTED BEFORE NV23 OR THEY WILL BE WASTED. STOP LEGACY PRECOMMITS ASAP TO AVOID FEES AND WASTED SEALING WORK. Please see more details in the next session.
```

Comment:
```suggestion
- ALL LEGACY PRECOMMITS MUST BE PROVECOMMITTED BEFORE NV23 OR THEY WILL BE WASTED. STOP LEGACY PRECOMMITS ASAP TO AVOID FEES AND WASTED SEALING WORK. Please see more details in the next section.
```